app/src/main/res/values/dimens.xml

```diff
@@ -214,13 +214,8 @@
     <dimen name="top_sites_text_margin_top">8dp</dimen>
 
     <!-- Credit Cards Fragment -->
-    <!-- The height of the "Add credit card" button in the Credit Cards fragment.
-         This should be the same value as search_engine_radio_button_height. -->
-    <dimen name="credit_cards_add_credit_card_button_height">48dp</dimen>
     <!-- The horizontal margin for the "Add credit card" text.
          This should be the same value as radio_button_padding_horizontal. -->
```

Comment:
Sorry, this should also be removed.

```suggestion
```

Reply:
No problem! I looked into the history and did not know exactly what it was referring to. Thanks for clearing things up!